src/libraries/System.Formats.Cbor/tests/CborDocument/System.Formats.Cbor.Tests.DataModel.fsproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <!-- NuGet restore issue with TargetFrameworks in f# projects: https://github.com/NuGet/Home/issues/10005 -->
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <!-- Public signing (performed under DotNetBuildSourceOnly) is broken for f#: https://github.com/dotnet/fsharp/issues/17451 -->
+    <SignAssembly Condition="'$(DotNetBuildSourceOnly)' == 'true'">false</SignAssembly>
```

Comment:
Wouldn't it make more sense to apply this to all fsproj files from a shared rule, and then remove it from the shared rule when the signing bug is fixed?

e.g. a rule that looks at the project file extension in a dirs.props or dirs.targets?

Reply:
If we want to apply this more general, we should probably do it in arcade `ProjectDefaults.props`.

This is a workaround for just this project (which seems to be the only affected).

@ViktorHofer @mmitche what is your preference?

Reply:
Agreed. Please make the change in Arcade, probably under Workaround.targets.